.config/ags/modules/.configuration/user_options.js

```diff
@@ -221,6 +221,11 @@ let configOptions = {
             'cycleTab': "Ctrl+Tab",
         }
     },
+    'bar': {
+        // Array of bar styles for each monitor.
+        // Example for four monitors: ["normal", "focus", "normal", "nothing"]
+        'monitors': ["normal"]
+    },
```

Comment:
let's have this in `bar.modes`, just in case we have another option for bar later on